srcCpp/RTIDDSImpl.cxx

```diff
@@ -3795,9 +3799,12 @@ double RTIDDSImpl_FlatData<T>::obtain_dds_serialize_time_cost_override(
         builder.add_latency_ping(0);
 
         // Add payload
+        
+      #ifndef RTI_CUSTOM_TYPE_FLATDATA
```

Comment:
Shouldn't we populate the custom type instead?

Otherwise having this metric when using custom types does not make much sense.